src/BotMan.php

```diff
@@ -497,7 +497,7 @@ protected function verifyServices()
      * @param string|array $recipients
      * @param DriverInterface|null $driver
      * @param array $additionalParameters
-     * @return Response
+     * @return array Response
```

Comment:
What about the Response[] syntax?
It help IDE to understand that it's an array of Response